repair_order_mass_validate/models/repair_order.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2025 Manuel Regidor <manuel.regidor@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class RepairOrder(models.Model):
+    _inherit = "repair.order"
+
+    def _get_complete_repairs_to_validate(self):
+        return self.filtered(lambda a: a.state == "draft")
+
+    def _get_complete_repairs_to_start(self):
+        return self.filtered(
+            lambda a: (
+                a.state == "confirmed" and a.invoice_method in ["none", "after_repair"]
+            )
+            or (a.state == "ready" and a.invoice_method == "b4repair")
+        )
+
+    def _get_complete_repairs_to_finish(self):
+        return self.filtered(
+            lambda a: (
+                a.state == "under_repair"
+                and a.invoice_method in ["none", "after_repair"]
+            )
+            or (a.invoice_method == "b4repair" and a.state == "under_repair")
+        )
+
+    def _get_complete_repairs_to_invoice(self):
+        return self.filtered(lambda a: a.state == "2binvoiced")
+
+    def complete_repair_validate(self):
+        draft_repair_orders = self._get_complete_repairs_to_validate()
+        for repair in draft_repair_orders:
+            repair.action_validate()
+
+    def complete_repair_start(self):
+        repair_orders_to_start = self._get_complete_repairs_to_start()
+        repair_orders_to_start.action_repair_start()
```

Comment:
Why not self._get_complete_repairs_to_start().action_repair_start()

Reply:
done